static/js/localStorage.js

```diff
@@ -13,20 +13,26 @@ const download = (filename, text) => {
 
 const savingData = () => {
```

Comment:
nitpick: 

Can we rename this to saveParametersToJson?

Reply:
+1